[2025-05-02T09:37:08.651+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-05-02T09:37:08.758+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: mysql_to_gcs.extract_customers manual__2025-05-02T09:37:03.654766+00:00 [queued]>
[2025-05-02T09:37:08.770+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: mysql_to_gcs.extract_customers manual__2025-05-02T09:37:03.654766+00:00 [queued]>
[2025-05-02T09:37:08.795+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-05-02T09:37:09.359+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): extract_customers> on 2025-05-02 09:37:03.654766+00:00
[2025-05-02T09:37:09.369+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1399) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-05-02T09:37:09.371+0000] {standard_task_runner.py:63} INFO - Started process 1401 to run task
[2025-05-02T09:37:09.371+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'mysql_to_gcs', 'extract_customers', 'manual__2025-05-02T09:37:03.654766+00:00', '--job-id', '42', '--raw', '--subdir', 'DAGS_FOLDER/mysql_to_gcs_dag.py', '--cfg-path', '/tmp/tmpak5lkj1z']
[2025-05-02T09:37:09.374+0000] {standard_task_runner.py:91} INFO - Job 42: Subtask extract_customers
[2025-05-02T09:37:10.327+0000] {task_command.py:426} INFO - Running <TaskInstance: mysql_to_gcs.extract_customers manual__2025-05-02T09:37:03.654766+00:00 [running]> on host dfbad22d9104
[2025-05-02T09:37:11.414+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='mysql_to_gcs' AIRFLOW_CTX_TASK_ID='extract_customers' AIRFLOW_CTX_EXECUTION_DATE='2025-05-02T09:37:03.654766+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-02T09:37:03.654766+00:00'
[2025-05-02T09:37:11.416+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-05-02T09:37:11.418+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/context.py:320: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2025-05-02T09:37:11.419+0000] {mysql_to_gcs_dag.py:19} INFO - Starting extraction for table: customers, timestamp: 20250502_093703
[2025-05-02T09:37:11.434+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2025-05-02T09:37:19.905+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/providers/common/sql/hooks/sql.py:263: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  return psql.read_sql(sql, con=conn, params=parameters, **kwargs)

[2025-05-02T09:37:20.586+0000] {mysql_to_gcs_dag.py:32} INFO - Saved CSV to: /tmp/mysql_extract/customers_data_20250502_093703.csv
[2025-05-02T09:37:20.596+0000] {connection.py:274} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2025-05-02T09:37:20.599+0000] {base.py:84} INFO - Using connection ID 'google_cloud_default' for task execution.
[2025-05-02T09:37:23.234+0000] {gcs.py:560} INFO - File /tmp/mysql_extract/customers_data_20250502_093703.csv uploaded to bronze_layer/customers_data_20250502_093703.csv in mysql-to-gcs-bronze-layer bucket
[2025-05-02T09:37:23.235+0000] {mysql_to_gcs_dag.py:41} INFO - Uploaded customers_data_20250502_093703.csv to GCS bucket.
[2025-05-02T09:37:23.236+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-05-02T09:37:23.238+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-05-02T09:37:23.250+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=mysql_to_gcs, task_id=extract_customers, run_id=manual__2025-05-02T09:37:03.654766+00:00, execution_date=20250502T093703, start_date=20250502T093708, end_date=20250502T093723
[2025-05-02T09:37:23.497+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-05-02T09:37:23.525+0000] {taskinstance.py:3503} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2025-05-02T09:37:23.713+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
